core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/ExposedService.java

```diff
@@ -47,7 +47,9 @@
     PROFILER_ADMIN("Profiler Admin", "PROFILER_ADMIN_AGENT", "PROFILER-ADMIN-API", "/profiler-admin/", false, 21700, 21700, true, true),
     PROFILER_METRICS("Profiler Metrics", "PROFILER_METRICS_AGENT", "PROFILER-METRICS-API", "/profiler-metrics/", false, 21800, 21800, true, true),
     PROFILER_SCHEDULER("Profiler Scheduler", "PROFILER_SCHEDULER_AGENT", "PROFILER-SCHEDULER-API", "/profiler-scheduler/", false, 21900, 21900, true, true),
-    DAS("Data Analytics Studio", "DAS_WEBAPP", "DAS", "/das/", true, 30800, 30800, false, true);
+    DAS("Data Analytics Studio", "DAS_WEBAPP", "DAS", "/das/", true, 30800, 30800, false, true),
+    KUDU("Kudu Master", "KUDU_MASTER", "KUDUUI", "/kuduui/", true, 8051, 8051, false, true),
+    KUDU_TABLET_SERVER("Kudu Tablet Server", "KUDU_TSERVER", "KUDUUIT", "/kuduui/", true, 8050, 8050, false, true);
```

Comment:
Is `KUDUUIT` a typo? Shouldn't it also be `KUDUUI`?

Reading more in the review, I see that at some level you're distinguishing between the master and tserver web UIs. Why? They behave identically apart from the difference in port number.

Reply:
@adembo I did this because Kudu Tablet Server should be available when KUDU_TSERVER is in the blueprint and Kudu Master should be visible when KUDU_MASTER is presented

Reply:
I see. Perhaps you could change the strings then so `KUDUUIT` doesn't look like a typo? Maybe something like `KUDUUI_MASTER` and `KUDUUI_TSERVER`?

Reply:
So  I would like to separate these two component KUDUUIT is the **t**ablet server